Adição de novos patrocinadores da comunidade

TiagoDanin opened the pull request:
## Contexto
Adição de novos patrocinadores da comunidade para o DevOpsDays Belém 2025, incluindo instituições de ensino e grupos de tecnologia locais, fortalecendo a representatividade do evento na região.

## O que foi feito?
- Adicionado AWS User Group Belém como patrocinador da comunidade
- Adicionado BelJogos como patrocinador da comunidade
- Adicionado CESUPA como patrocinador da comunidade

TiagoDanin committed:
feat: add new sponsors for DevOpsDays Belém 2025

- Introduced new sponsor logos for AWS User Group Belém, BelJogos, and CESUPA.
- Updated the sponsors data structure to include the new entries with relevant details and logo paths.

thalisonwilker commented:
## Corrigir links quebrados

Caminho | Link quebrado | link correto
----------| ----------|------------
Home/Patrocinadores/Comunidade|https://www.beljogos.com.br/|http://www.beljogos.com.br/ (SEM HTTPS)

## Mover o patrocinador CESUPA para APOIO

Caminho | Patrocinador|Nível incorreto | Nível correto
----------| ----------|------------|----------
Home/Patrocinadores/Comunidade|CESUPA|Comunidade|Apoio

TiagoDanin committed:
feat: add new sponsors for DevOpsDays Belém 2025

- Added Techlead and Inteceleri as new sponsors with their respective details and logo paths.
- Updated the URL for BelJogos to use the correct protocol.
